controllers/comments.controller.js

```diff
@@ -7,4 +8,17 @@ getCommentsById(article_id)
     res.status(200).send({comments:comments})
 })
 .catch(next)
+}
+
+exports.sendComment = (req, res, next) => {
+const {article_id} = req.params
+getArticleById(article_id)
+.then(({article_id}) => {
+        const newComment = req.body
+        postComment(newComment, article_id)
+        .then(({body}) => {
+        res.status(201).send({comment:body})  
```

Comment:
you're using `getArticleById` here to check if the article exists first before making a POST query. Take a look at the `seed.js` file, where the _comments_ table is created it has some `FOREIGN KEY` columns. 
Foreign key constraints  protect against new data being inserted that cannot link to other tables. This mean that when making an INSERT query, if those values don't exist on the other tables PSQL will reject it automatically. So in this instance you don't have to use `getArticleById` to check if the article exists